### ISSUE TITLE
Can't import plotly 2.0.3

### ISSUE THREAD

**Issue by bnaul** [plotly/plotly.py#706]
Python 2.7 or 3.5:
```
$ python -c 'import plotly'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/bnaul/miniconda3/lib/python3.5/site-packages/plotly/__init__.py", line 31, in <module>
    from plotly import (plotly, graph_objs, grid_objs, tools, utils, session,
  File "/home/bnaul/miniconda3/lib/python3.5/site-packages/plotly/plotly/__init__.py", line 10, in <module>
    from . plotly import (
  File "/home/bnaul/miniconda3/lib/python3.5/site-packages/plotly/plotly/plotly.py", line 33, in <module>
    from plotly.dashboard_objs import dashboard_objs as dashboard
ImportError: No module named 'plotly.dashboard_objs'
```
Seems weird that there's no 2.0.3 tag on GitHub but I'm guessing something just went wrong with the PyPI upload?

**Comment by cldougl** [comment]
@bnaul thanks for letting us know, this should be fixed shortly and re-released.

**Comment by chohner** [comment]
Seems to be still going on in `2.0.4` :/

```
In [1]: import plotly
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-c27a4132ad2e> in <module>()
----> 1 import plotly

/Users/chohner/Downloads/plotly-2.0.4/plotly/__init__.py in <module>()
     29 from __future__ import absolute_import
     30
---> 31 from plotly import (plotly, dashboard_objs, graph_objs, grid_objs, tools,
     32                     utils, session, offline, colors)
     33 from plotly.version import __version__

/Users/chohner/Downloads/plotly-2.0.4/plotly/plotly/__init__.py in <module>()
      8
      9 """
---> 10 from . plotly import (
     11     sign_in,
     12     update_plot_options,

/Users/chohner/Downloads/plotly-2.0.4/plotly/plotly/plotly.py in <module>()
     31 from plotly.plotly import chunked_requests
     32 from plotly.grid_objs import Grid, Column
---> 33 from plotly.dashboard_objs import dashboard_objs as dashboard
     34
     35 # This is imported like this for backwards compat. Careful if changing.

ImportError: No module named 'plotly.dashboard_objs'

**Comment by chohner** [comment]
Fixed in 2.0.5

**Comment by bnaul** [comment]
Same! Glad to see this is being properly tested now as well :+1:.
